### ISSUE TITLE
Not able to capture image

### ISSUE THREAD

**Issue by puneetamb** [rokuality/rokuality-js#11]
Hi Team,

getImage() is not working.

Error:
```

    ENOENT: no such file or directory, open '/var/folders/13/t9q7wfjj0gz907xtrk5vgvt80000gn/T/com.rokuality/4pvqbp3fzymxx9tszule0ylgbqkl1zhmrpnnjc69.png'

      110 |
      111 |       var filePath = rokualityTempDir + path.sep + (await this.generateRandomString(40)) + fileExtension;
    > 112 |       fs.writeFileSync(filePath, fileData);
      113 |       return filePath;
      114 |     }
      115 |   }, {
      
      at JSONUtils.convertToFile (../build/utils/FileToStringUtils.js:112:10)
      at async RokuScreen.getImage (../build/driver/Screen.js:96:14)
      at async Object.<anonymous> (roku.test.js:109:5)
```

**Comment by rokualitysupport** [comment]
Can you check if this directory exists on your machine? /var/folders/13/t9q7wfjj0gz907xtrk5vgvt80000gn/T/com.rokuality
It looks like there may be an issue creating the temp directory to save file assets on your machine. We will investigate but in the short term you can manually create that directory and then this should work

**Comment by rokualitysupport** [comment]
Please try new Rokuality version 1.3.2 available at https://www.rokuality.com/download which on start will help manage those local user temp directories and should prevent this issue.

**Comment by rokualitysupport** [comment]
This is now fixed client side in latest js version 1.5.2
